src/CMakeLists.txt

```diff
@@ -123,20 +127,23 @@ set(OPCUABRIDGE_HEADERS opcuabridge/boostarch.h
                         opcuabridge/metadatafile.h
                         opcuabridge/metadatafiles.h
                         opcuabridge/opcuabridge.h
+                        opcuabridge/opcuabridgeclient.h
+                        opcuabridge/opcuabridgeserver.h
                         opcuabridge/signature.h
                         opcuabridge/signed.h
                         opcuabridge/utility.h
-                        opcuabridge/versionreport.h)
-
+                        opcuabridge/versionreport.h
+                        uptane/opcuasecondary.h)
```

Comment:
This probably lives in a CMakeLists.txt under `src/opcuabridge`, but there is clearly some interdependency with `src/uptane` at the moment.  I think we can live with this for now, but this is something we will have to look at in the future.  Probably the answer is to make `src/uptane` purely an implementation of the Uptane spec and move the transport and installation related parts somewhere else.